src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,61 @@
+package lotto.domain;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public record Lotto(Set<LottoNumber> numbers) {
+    private static final int LOTTO_NUMBER_COUNT = 6;
+
+    public Lotto(List<LottoNumber> inputs) {
+        this(toValidatedSet(inputs));
+    }
+
+    private static Set<LottoNumber> toValidatedSet(List<LottoNumber> inputs) {
```

Comment:
```suggestion
    private static Set<LottoNumber> toSet(List<LottoNumber> inputs) {
```
유효성 체크는 당연한 부분이라 굳이 메서드 이름에 포함할 필요가 있을까?